common/GNUmakefile

```diff
@@ -147,6 +147,18 @@ ifeq ($(TARGET),uefi-riscv64)
         -D__riscv64
 endif
 
+ifeq ($(TARGET),uefi-loongarch64)
+    override CFLAGS_FOR_TARGET += \
+        -fPIE \
+        -fshort-wchar
```

Comment:
I assume no flag to disable FPU/SIMD is necessary?

Reply:
Also, is a flag like `--no-relax` not needed? I see `.sbss` and `.sdata` being defined in the linker script, so I am not sure if that is an oversight or not.